media_kit/lib/src/player/libmpv/player/real.dart

```diff
@@ -1172,16 +1176,24 @@ class libmpvPlayer extends PlatformPlayer {
       if (prop.ref.name.cast<Utf8>().toDartString() == 'pause' &&
           prop.ref.format == generated.mpv_format.MPV_FORMAT_FLAG) {
         if (isPlayingStateChangeAllowed) {
-          final playing = prop.ref.data.cast<Int8>().value != 1;
+          final playing = prop.ref.data.cast<Int8>().value == 0;
           state = state.copyWith(playing: playing);
           if (!playingController.isClosed) {
             playingController.add(playing);
           }
         }
       }
+      if (prop.ref.name.cast<Utf8>().toDartString() == 'core-idle' &&
+          prop.ref.format == generated.mpv_format.MPV_FORMAT_FLAG) {
+        final buffering = prop.ref.data.cast<Int8>().value == 1;
```

Comment:
should this be `and playing`?

since if this condition doesn't exist it will buffering is true when paused?

Reply:
I noticed the issue & resolved it in #250 tree. The `&& playing` had some race issue which made it fail on runner.

https://github.com/alexmercerind/media_kit/blob/e2588461bd8c0a378379f8af911587cb3463e552/media_kit/lib/src/player/libmpv/player/real.dart#L1242-L1255